docs/pages/custom-builds/schema.mdx

```diff
@@ -1300,6 +1300,153 @@ build:
   href="https://github.com/expo/eas-build/blob/main/packages/build-tools/src/steps/functions/uploadArtifact.ts"
 />
 
+### Using built-in EAS functions to build your app
+
+Using the built-in EAS functions you can recreate the default EAS Build workflow for different build types.
+
+For example, to create that creates internal distribution build for Android and a simulator build for iOS you can use the following configuration:
```

Comment:
```suggestion
For example, to trigger a build that creates internal distribution build for Android and a simulator build for iOS you can use the following configuration:
```

I think the above sentence is missing the context of creating a build. It is not clear by "to create...". The user has to create "what" in order to create an internal distribution.

I assume it's a build.